utilities_common/cli.py

```diff
@@ -269,6 +269,112 @@ def is_port_vlan_member(config_db, port, vlan):
 
     return False
 
+
+def vlan_range_list(ctx, vid_range: str) -> list:
+
+    vid1, vid2 = map(int, vid_range.split("-"))
+
+    if vid1 == 1 or vid2 == 1:
+        ctx.fail("Vlan1 is default vlan")
+
+    if vid1 >= vid2:
+        ctx.fail("{} is greater than {}. List cannot be generated".format(vid1,vid2))
+
+    if is_vlanid_in_range(vid1) and is_vlanid_in_range(vid2):
+        return list(range(vid1, vid2+1))
+    else:
+        ctx.fail("Invalid VLAN ID must be in (2-4094)")
```

Comment:
>ctx.fail("Invalid VLAN ID must be in (2-4094)") [](http://example.com/codeflow?start=8&length=47)

Please check coverage result [https://dev.azure.com/mssonic/build/_build/results?buildId=513942&view=codecoverage-tab](https://dev.azure.com/mssonic/build/_build/results?buildId=513942&view=codecoverage-tab). Since these functions are intended to be reused as library functions, please cover it 100%.